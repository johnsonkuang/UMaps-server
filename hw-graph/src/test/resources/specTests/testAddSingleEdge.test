# Test Add Single Edge
# tests the creation of a single edge connecting two nodes

# Create a graph
CreateGraph graph1

# Add a pair of nodes to the graph
AddNode graph1 n1
AddNode graph1 n2

# Add a edge connecting n1 and n2
AddEdge graph1 n1 n2 A