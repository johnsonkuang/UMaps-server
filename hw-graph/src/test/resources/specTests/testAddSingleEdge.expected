# Test Add Single Edge
# tests the creation of a single edge connecting two nodes

# Create a graph
created graph graph1

# Add a pair of nodes to the graph
added node n1 to graph1
added node n2 to graph1

# Add a edge connecting n1 and n2
added edge A from n1 to n2 in graph1